src/js/layerlist/LayerList.js

```diff
@@ -18,15 +18,19 @@ L.DNC.LayerList = L.Control.extend({
         layerContainerId: 'dropzone'
     },
 
-    initialize: function (options) {
+    initialize: function (map, options) {
```

Comment:
Pass in `map` object on init rather than using `.addTo()`. More in line with the way that `L.DNC.DropZone` is initialized.